### PR TITLE
Remove apostrophe from HTML tags

### PR DESCRIPTION
**Pull request by zarahzachz**
No known associated issue number. Grammatical fix.

Changes proposed in this pull request:

Removed improper use of apostrophe from HTML tags in content. The 's' was added to these tags to indicate multiples of said tag, not possessiveness.

### CODE REVIEW

**Review by estelle — APPROVED** [review]
thank's. (just kidding)

File: src/site/content/en/accessible/semantics-and-screen-readers/index.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ properties:
 - A **state** (optional)
 
 An element's **role** describes its type, i.e. "button," "input," or even just
-"group" for things like `div`'s and `span`'s.
+"group" for things like `div`s and `span`s.
```
Comment (sofiayem) [inline src/site/content/en/accessible/semantics-and-screen-readers/index.md:77]:
```suggestion
"group" for things like `div` and `span` elements.
```

File: src/site/content/en/accessible/semantics-and-screen-readers/index.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ to the user, it is often doing so by walking this accessibility tree.
 Browsers will often remove semantically uninteresting nodes like `div` and
 `span` from the accessibility tree, especially if they're just being used to
 position their children with CSS. For instance, if you have a `button` nested
-inside of 5 `div`'s, the browser may prune out some of the `div`'s in the middle
+inside of 5 `div`s, the browser may prune out some of the `div`s in the middle
```
Comment (sofiayem) [inline src/site/content/en/accessible/semantics-and-screen-readers/index.md:106]:
```suggestion
inside of 5 `div` elements, the browser may prune out some of the `div` elements in the middle
```

**Review by sofiayem — APPROVED** [review]

**Comment by sofiayem** [comment]
@zarahzachz Thank you for spotting the incorrect use of possessives instead of plurals. However, there's a [guideline](https://developers.google.com/style/code-in-text#grammatical-treatment-of-code-elements) to avoid treating in-line code as parts of speech. Fixed it, so now it's 100% correct :)

**Pull request merged**